packages/pagination/src/Pagination/PaginationRangeView.spec.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import PaginationRangeView from './PaginationRangeView';
+
+describe('PaginationRangeView', () => {
+  describe('a11y', () => {
+    test('does not have basic accessibility issues', async () => {
+      const { container } = render(
+        <PaginationRangeView
+          itemsPerPage={10}
+          currentPage={1}
+          numTotalItems={100}
+        />,
+      );
+      const results = await axe(container);
+      expect(results).toHaveNoViolations();
+    });
+  });
+
+  describe('item range display', () => {
+    test('renders "of many" when numTotalItems is undefined', () => {
+      const { getByTestId } = render(
+        <PaginationRangeView itemsPerPage={10} currentPage={1} />,
+      );
+      expect(getByTestId('lg-pagination-item-range').textContent).toBe(
+        '1 - 10 of many',
+      );
```

Comment:
Nit: we could make these tests a bit DRYer by using variables
```suggestion
      const itemsPerPage = 10;
      const currentPage = 1;
      const { getByTestId } = render(
        <PaginationRangeView itemsPerPage={itemsPerPage} currentPage={currentPage} />,
      );
      expect(getByTestId('lg-pagination-item-range').textContent).toBe(
        `${currentPage} - ${itemsPerPage} of many',
      );
```

Reply:
Sounds good. This isn't a hill I'll die on at all, and in this case DRY / not DRY doesn't make bit difference. I will only say that my opinion is that I think its dangerous to set a rule "no DRYness in tests," it really should depend on the test and what's being tested. As we saw in the test with the constant - whether we make that test DRY or not DRY, impacts what we're actually testing. In this case it doesn't matter though, so will resolve.